src/main/java/school/lemon/changerequest/java/introduction/hw2/Task8.java

```diff
@@ -1,8 +1,23 @@
 package school.lemon.changerequest.java.introduction.hw2;
 
 public class Task8 {
-
     public static int[] evenOdd(int[] arr) {
-        return null;
+        if (arr == null) {
+            return null;
+        }
+        int result[] = new int[arr.length];
+        int a = 0;
+        int b = 0;
+        for (int Odd : arr) {
```

Comment:
Please follow java naming convention. Variables should be in camelCase. 
Also not sure `Odd` is correct name.